lib/db/pokemon_grabber.py

```diff
@@ -0,0 +1,70 @@
+import requests
+
```

Comment:
I think this would be better in a class as then you could have self.api = "{apilink}" rather than having to redefine it in multiple sections.

Reply:
After going through this section, 'apilink' is unique in all the functions. We will keep it how it is for now.